projects/step-core/src/lib/components/toast-container/toast-container.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { ToastService } from '../../services/toast.service';
+import { NotificationAction } from '../../shared/toast-action.interface';
+
+@Component({
+  selector: 'step-toast-container',
+  templateUrl: './toast-container.component.html',
+  styleUrls: ['./toast-container.component.scss'],
+})
+export class ToastContainerComponent implements OnInit {
+  toasts: { message: string; actions?: NotificationAction[]; duration?: number }[] = [];
+
+  constructor(private toastService: ToastService) {}
+
+  ngOnInit(): void {
+    this.toastService.toastMessages.subscribe((toasts) => {
+      this.toasts = toasts;
+    });
+  }
```

Comment:
Please, check the comment below in ToastService. With signal usage the code of current component could look like this
```suggestion
protected readonly _toasts = inject(ToastService).toastMessages;
```